### PR TITLE
Add my thanks

### PR DESCRIPTION
**Pull request by firdavsDev**
* Add my thanks
* Fix all replace Uzbekistan with Uzbekistan 🇺🇿 (with flag)

* [X] I would also like a **Thank you PHP** badge on my GitHub profile.

### CODE REVIEW

**Comment by firdavsDev** [comment]
ops, what's wrong on sorting it? :(

**Comment by firdavsDev** [comment]
@petk could you help me, please

**Comment by petk** [comment]
Of course. Thanks and welcome here. Can we perhaps just change those flags a bit. There are now two flags on some place.

File: README.md [hunk 1]
```diff
@@ -381,9 +382,9 @@ Signed,
 - Sergio de Candelario ([@sdecandelario](https://github.com/sdecandelario)), Software Engineer, Barcelona, Catalonia, Spain :)
 - Serhat Aksakal ([@fanksin](https://github.com/fanksin)), ❤️ PHP since 2015, 🇹🇷 Türkiye
 - Shahadat Hossain Hridoy ([@HridoyHazard](https://github.com/HridoyHazard)), Mern Stack Developer, Bangladesh
-- Shakhobiddin Urinbayev ([@shokhaa](https://github.com/shokhaa)), PHP Backend Developer, Uzbekistan
+- Shakhobiddin Urinbayev ([@shokhaa](https://github.com/shokhaa)), PHP Backend Developer, Uzbekistan 🇺🇿
 - Shakil Ahmmed ([@ShakilAhmmed](https://github.com/ShakilAhmmed)), Software Engineer, 🇧🇩 Bangladesh
-- Shukrullo Turgunov ([@Vodiylik](https://github.com/vodiylik)), Software Engineer, 🇺🇿 Uzbekistan
+- Shukrullo Turgunov ([@Vodiylik](https://github.com/vodiylik)), Software Engineer, 🇺🇿 Uzbekistan 🇺🇿
```
Comment (petk) [inline README.md:387]:
```suggestion
- Shukrullo Turgunov ([@Vodiylik](https://github.com/vodiylik)), Software Engineer, Uzbekistan 🇺🇿
```

**Comment by petk** [comment]
Added via https://github.com/thank-you-php/thank-you-php/commit/ac75df7ec9aa1208c858eaec2aed8a994f333c48

Thanks @firdavsDev

**Pull request closed without merging**